libbeat/docs/index.asciidoc

```diff
@@ -28,6 +28,8 @@ include::./dashboards.asciidoc[]
 
 include::./newbeat.asciidoc[]
 
+include::./config-file-format.asciidoc[]
```

Comment:
We can add it to the Platform ref, too...just need to make the content generic enough.